hack/00-osd-managed-cluster-config.selectorsyncset.yaml.tmpl

```
@@ -1192,7 +1192,7 @@ objects:
               operator: DoesNotExist
         quota:
           hard:
-            services.loadbalancers: 2
+            services.loadbalancers: '2'
```

```suggestion
            services.loadbalancers: "2"
```

For consistency only, double quotes instead of single quotes.

---

Very strange. The template has double quotes, so I don't understand why the auto-generated file has single quotes.

---

@jewzaam I'm not going to apply your suggestion because in the rendered yaml file each other string has single quotes around it. Elsewhere, in non-generated files, we use double quotes, so it seems we have a difference of opinion compared to Python's YAML libraries.

---

Ah, fair enough.  Didn't notice it was the generated template. :+1: